CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ if (APPLE)
     set(CMAKE_MACOSX_RPATH TRUE)
 endif()
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 20)
```

Comment:
You can bump this to 17 if it's really needed to compile any of the replacements.